samples/bluetooth/hci_ipc/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _bluetooth-hci-ipc-sample:
+.. zephyr:code-sample:: bluetooth_hci_ipc
+   :name: HCI IPC
+   :relevant-api: hci_raw bluetooth
 
-Bluetooth: HCI IPC
-##################
+   Expose Zephyr Bluetooth controller to another device or CPU using the IPC subsystem.
```

Comment:
It can be any configured bluetooth controller. eg, even one connected over UART

Reply:
same comment for the other hci_xx samples

Reply:
I took the wording from the "Overview" section of most HCI samples, so I guess they need to be fixed?
https://docs.zephyrproject.org/latest/samples/bluetooth/hci_spi/README.html#overview

Reply:
oh yeah then they need to fixed, yes

Reply:
should be addressed both in the new descriptions and legacy "Overview" sections